### ISSUE TITLE
Pattern fill not working with latest react-native

### ISSUE THREAD

**Issue by ak-spotter** [software-mansion/react-native-svg#2437]
### Description

Pattern fill is working in older versions of react-native. But if you create a new react-native project, it is not showing up.

Here's an example-
```js
import {Path, Pattern} from 'react-native-svg';

const App = () => {
  const MyPattern = () => {
      return (
        <Pattern
          id="DiagonalLines"
          patternUnits="userSpaceOnUse"
          x="0"
          y="0"
          width={60}
          height={14}
          viewBox="0 0 40 20">
          <Path
            d={`M ${-1220} ${38} L ${465} ${0}`}
            stroke={'white'}
            strokeWidth={1.5}
            transform={{rotation: '-16'}}
          />
        </Pattern>
      );
    };
    
    const data=[
    {
      value: 110,
      barBackgroundPattern: MyPattern,
      patternId: 'DiagonalLines',
      frontColor: '#4444dd',
      spacing: 8,
    },
    {value: 130},
    {
      value: 120,
      barBackgroundPattern: MyPattern,
      patternId: 'DiagonalLines',
      frontColor: '#4444dd',
      spacing: 8,
    },
    {value: 160},
    {
      value: 190,
      barBackgroundPattern: MyPattern,
      patternId: 'DiagonalLines',
      frontColor: '#4444dd',
      spacing: 8,
    }]
    
    return <BarChart data={dtt} barWidth={40} spacing={30} />
 }
```
Below is the desired output which was produced  by the above code-
![image](https://github.com/user-attachments/assets/eef37ad3-10e5-492c-84ef-39126cb18bc8)

But with latest react-native (i.e- version- 0.75), the pattern is not appearing.

### Steps to reproduce

1. Create a new react-native app using react native cli using the below command-
```
npx @react-native-community/cli@latest init AwesomeProject
```
2. Install the below-
```
npm install react-native-gifted-charts react-native-linear-gradient react-native-svg
```
Then Reander a bar chart with a pattern inside

### Snack or a link to a repository

https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts

### SVG version

15.6.0

### React Native version

0.75.2

### Platforms

Android

### JavaScript runtime

None

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Acknowledgements

Yes

**Comment by bohdanprog** [comment]
Hello @ak-spotter,
When I tried to reproduce that issue I noticed that even if I ran my app in react-native 0.73.1 version. I still have something like that: 
![Screenshot_1725285827](https://github.com/user-attachments/assets/5ab2ae43-ab7d-4433-8898-be107eaa3f64)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @bohdanprog Thanks for testing the issue.
You are seeing the issue even in older versions of react-native (like `0.73.1`). This is because you used this version, but the **`react-native-gifted-charts`** library is still using the latest react native version. If I degrade the react-native version in the _react-native-gifted-charts_ library, it starts working fine.

I have found the reason why the pattern is not getting displayed. It is because I was using `stroke='transparent'` in the _react-native-gifted-charts_ library. Replacing it with `stroke='none'` fixes the issue.
`stroke='transparent'` is not working when using the latest version of react-native.

We can close this issue, as I have found the solution.
